model/model.go

```diff
@@ -88,6 +90,31 @@ func (c *ColumnInfo) IsGenerated() bool {
 	return len(c.GeneratedExprString) != 0
 }
 
+// SetDefaultValue sets the default value.
+func (c *ColumnInfo) SetDefaultValue(value interface{}) error {
+	c.DefaultValue = value
+	if c.Tp == mysql.TypeBit {
+		// For mysql.TypeBit type, the default value storage format must be a string.
+		// Other value such as int must convert to string format first.
+		if v, ok := value.(string); ok {
+			c.DefaultValueBit = []byte(v)
+			return nil
+		}
+		return types.ErrInvalidDefault.GenByArgs(c.Name)
+	}
+	return nil
+}
+
+// GetDefaultValue gets the default value of the column.
+// Default value use to stored in DefaultValue field, but now,
+// bit type default value will store in DefaultValueBit for fix bit default value decode/encode bug.
+func (c *ColumnInfo) GetDefaultValue() interface{} {
```

Comment:
Do we need to change all "c.DefaultValue" to "GetDefaultValue()"?

Reply:
Do we need to change "if column.DefaultValue != nil" in column.go?

Reply:
"if column.DefaultValue != nil" in column.go, the `ColumnInfo` in column.go is not the `ColumnInfo` in model.go

Reply:
Do we need to replace "c.DefaultValue = types.ZeroDatetimeStr" with "SetDefaultValue()" in ddl_api.go?

Reply:
oh...
ye...

Reply:
done.